rpcs3/Emu/Cell/lv2/sys_game.cpp

```diff
@@ -3,17 +3,17 @@
 
```

Comment:
Not sure why you moved the class instead making a new one.

Reply:
Updated